### PR TITLE
NONE: give support access to api/hash endpoint

### PR DESCRIPTION
**Pull request by rachellerathbone**
**What's in this PR?**
Update to our poco config

**Why**
So support can access our hash endpoint

**Whats Next?**
Grant access to splunk logs for support

### CODE REVIEW

**Comment by atrigueiro** [comment]
Nice, tests?

File: etc/poco/bundle/main-test.json [hunk 1]
```diff
@@ -87,6 +87,14 @@
 			"mechanism": "asap",
 			"principals": ["micros/manager-service"],
 			"allowed": true
+		},
+		{
+			"name": "Allow support team to call API endpoints",
+			"path": "/api/hars",
```
Comment (atrigueiro) [inline etc/poco/bundle/main-test.json:93]: Did this test pass or fail? Shouldn't that be `/api/hash` ?

File: etc/poco/bundle/main-test.json [hunk 2]
```diff
@@ -87,6 +87,14 @@
...
+			"method": "GET",
+			"mechanism": "open",
+			"principals": [],
```
Comment (atrigueiro) [inline etc/poco/bundle/main-test.json:96]: You should add support team here as the only allowed principal, and then add another test where allowed: false for a random test principal

File: etc/poco/README.md [hunk 3]
```diff
@@ -37,18 +37,20 @@ Refer to [Testing](https://developer.atlassian.com/platform/poco/policies/workfl
 Poco bundles can be manually published to dev and staging environments using atlas-cli. Note that this will apply
 to the deployment stack already in service without needing to redeploy.
 
+### Publish into dev
 ```shell
```
Comment (rachellerathbone) [inline etc/poco/README.md:41]: Separated these out so you can use the copy button without getting a bunch of stuff you don't need to dev/stg

**Comment by rachellerathbone** [comment]
Passing tests
Screenshot 2023-05-11 at 1 59 37 pm

**Review by atrigueiro — APPROVED** [review]

**Review by atrigueiro — APPROVED** [review]

**Pull request merged**